rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,172 @@
+- An RFC is required if a technical initiative (refactoring, major architectural change) impacts the community.
+- A RFC is required for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).  
+
+### Revising RFCs
+
+Where necessary, existing RFCs can be revised using the same process (a proposed change submitted as a RFC pull request and subject to community review and approval). **Major changes should result in a new RFC.**
+
+### What are the roles in the RFC process?
+
+- **Authors** are DCP community members.
+
+- **Shepherds** are members of *DCP PM* assigned to assist the **Author(s)** and ensure that the RFC progresses to closure.
+
+- **Approvers** for governance RFCs are the Project Leads and Product Owners from *DCP PM*.
+
+- **Approvers** for software RFCs are the Technical Leads from *DCP Architecture*. 
+
+- **Reviewers** are DCP community members, including both software developers and users.
+
+- **Science Reviewer** is the *DCP Science PM* in coordination with *HCA Science Governance*.
+
+## Detailed Design
+
+### Proposing RFCs
+
+#### **Authors**:
+
+*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+
+  - Fork the HumanCellAtlas dcp-community repository
+  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
+  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
+  
+  - Fill in the RFC with attention to detail
+
+  - Submit a pull request.
+  
+      For software RFCs:
+      - Add the "rfc-proposed" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+
+      For governance RFCs:
+        
+      - Add the "rfc-proposed" and the "Governance" labels. 
+
+  - Share a link to the RFC pull request for community review on the HumanCellAtlas **#dcp** slack channel
+
+#### DCP PM:
+- Assign a **Shepherd** by completing the *Shepherd* section in the RFC template and pushing the commit. The **Shepherd** guides the RFC and its **Author(s)** through the process.
+- If there is a *Scientific "guardrails"* section in the RFC, assign the **Science Reviewer** as a reviewer of the pull request and add the "science-review-required" label.
+
+#### Shepherd:
+- Include the RFC submission in *This week in DCP* status update
```

Comment:
Link to this...

Reply:
Clarifying - Do you mean _Link to this RFC submission in ..._

Reply:
My understanding is "This Week in DCP" is a slack channel so reference that here...

Reply:
Rust and Kubernetes use a combination of a web site (similar to a blog) and a mail chimp list for subscribers. Science Governance may prefer an email message over monitoring a slack channel?